fastlane/metadata/default/name.txt

```diff
@@ -0,0 +1 @@
+WordPress – Website Builder
```

Comment:
Reminder: `deliver` uses the files in the `default` folder as a fallback if it finds no localized value.